docs/sdkx-react-native/support-tools.mdx

```diff
@@ -0,0 +1,154 @@
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Helpshift APIs {#support-tools}
+
+<Intro>
+
+Helpshift provides a way to use each functional element of the SDK separately based on your requirements.
+You can call only specific screens from the SDK based on the functionality you want to expose in your app.
+
+</Intro>
+
+<Admonition type="info" title="Note">
+
+All the public APIs in the SDK should be called after initializing the SDK via [Helpshift.install() API](/sdkx-react-native/getting-started#start-using)
+
+</Admonition>
+
+## Integrating Contact Us & In App Messaging {#conversation-view}
+
+<Centered width="major">
+
+![newConversation.png](/static/books/sdkx_android/show_conversation.png "newConversation.png")
+
+</Centered>
+
+You can use this API call to allow a user to directly send feedback or start a new conversation without having to first view FAQs.
+
+This API call will show the conversation screen. Once a conversation is started, it will continue until it is resolved or rejected by the agent on Helpshift Dashboard.
+
+To configure SDK, you can pass various config options as a parameter. Check out more configuration options [here].(/sdkx-react-native/sdk-configuration)
+
+Example :
+
+```
+//Create your config here.
+const config = {};  
+
+// pass the config in the api
+showConversation(config);
+
+```
+
+## Integrating FAQs {#faqs-view}
+
+<Centered >
+
+![showFAQs.png](/static/books/sdkx_android/show_faqs.png "showFAQs.png")
+
+</Centered>
+
+You can use this API to provide a way for the user to invoke the purpose-built help/support section in your app. This is the easiest approach to enable help in your app as it bundles all the capabilities of the Helpshift SDK in a simple and intuitive interface. You can wire this API call to a **"Help"** or **"Support"** action in your app.
+
+To configure the FAQs screen, you can pass various config options as a parameter. Check out more config options [here](/sdkx-react-native/sdk-configuration)
+
+Example :
+
+```
+showFAQsWithConfig({
+  faq: <FAQ_SECTION_ID> 
+});
+
+```
+## Showing a Particular FAQ Section {#faq-section-view}
+
+<Centered >
+
+![showFAQSection.png](/static/books/sdkx_android/show_section.png "showFAQSection.png")
+
+</Centered>
+
+You can use this API to invoke a particular section of your FAQs with its FAQ section publish-id
+
+This feature works like a permalink for displaying specific FAQ sections as context-sensitive help in your app. For example, if your app requires the user to login to using email, Facebook, and Twitter, you could link a help action on the login screen that can link to the Helpshift FAQ section called "Login help" which has several questions related to login methods.
+
+Example :
+```
+showFAQSectionWithConfig(
+  <FAQ_SECTION_PUBLISH_ID>, {
+    sectionFaq: <DEFAULT_SECTION_ID>
```

Comment:
{
    sectionFaq: <DEFAULT_SECTION_ID>
  }
  
  This might be confusing just put config here.

Reply:
updated.